src/components/Select/Select.js

```diff
@@ -76,23 +74,51 @@ const Select = React.forwardRef(
             {children}
           </select>
           {componentsX ? (
-            <ChevronDownGlyph
-              aria-hidden={true}
-              aria-label={iconDescription}
-              alt={iconDescription}
-              className={`${prefix}--select__arrow`}
-              name="chevron--down"
-            />
+            <ChevronDownGlyph className={`${prefix}--select__arrow`}>
+              <title>{iconDescription}</title>
+            </ChevronDownGlyph>
           ) : (
             <Icon
               icon={iconCaretDown}
               className={`${prefix}--select__arrow`}
               description={iconDescription}
             />
           )}
-          {!componentsX && helper}
-          {componentsX && inline && helper}
-          {error}
+          {componentsX && invalid && (
+            <WarningFilled16 className={`${prefix}--select__invalid-icon`} />
+          )}
+        </>
+      );
+    })();
+    return (
+      <div className={`${prefix}--form-item`}>
+        <div className={selectClasses}>
+          <label htmlFor={id} className={labelClasses}>
+            {labelText}
+          </label>
+          {componentsX && !inline && helper}
```

Comment:
The markup change seems to affect `v9`. Could you please double-check `v9` is not broken?

Reply:
it should be fixed now @asudoh

Reply:
OK thank you for adding that back! Actually `v9` topic came up as the helper used to be next icon whereas it's next to `<label>` in this PR. Wanted to make sure such move-around doesn't negatively affecting `v9`. Thanks!

Reply:
@emyarod You are right, I may have been seeing a stale commit, my apologies!